app/helpers/application_helper/button/cloud_volume_new.rb

```diff
@@ -0,0 +1,18 @@
+class ApplicationHelper::Button::CloudVolumeNew < ApplicationHelper::Button::Basic
+  def calculate_properties
+    super
+    if disabled?
+      self[:title] = _("No cloud providers support creating cloud volumes.")
+    end
+  end
+
+  def disabled?
+    ems_clouds = EmsCloud.all
+    # if any connected provider supports this action,
+    # we can enable the button.
+    ems_clouds.each do |ems|
+      return false if CloudVolume.class_by_ems(ems).supports_create?
+    end
+    true
```

Comment:
second thought. we aren't looking at `Rbac` or anything here.
Lets just use `none?` and be done with it.

I'll ping @martinpovolny and mention this pattern to him and see what he wants to do going forward. He seems to have lots of opinions on this specific code pattern